src/Frontend/Modules/Tags/Tests/Engine/ModelTest.php

```diff
@@ -173,6 +173,6 @@ private function assertModuleTags($items): void
         self::assertArrayHasKey('name', $items);
         self::assertArrayHasKey('label', $items);
         self::assertArrayHasKey('items', $items);
```

Comment:
But if we change all the static methods to instance methods for phpunit, we should change them all, not leave the tests in a half state of mixed statics with instance methods...